### PR TITLE
docs: Updated supported APIs, included IAM and CloudFront APIs

### PR DESCRIPTION
**Pull request by JigarJoshi**

### CODE REVIEW

File: docs/api/s3/index.md [hunk 1]
```diff
@@ -85,6 +85,31 @@ status within Tigris.
 | [UploadPart](https://docs.aws.amazon.com/AmazonS3/latest/API/API_UploadPart.html)                                 | Yes                 |
 | [UploadPartCopy](https://docs.aws.amazon.com/AmazonS3/latest/API/API_UploadPartCopy.html)                         | Yes                 |
 
+## IAM APIs
```
Comment (ovaistariq) [inline docs/api/s3/index.md:88]: Could you keep this as a table like above so it looks consistent?

File: docs/api/s3/index.md [hunk 2]
```diff
@@ -85,6 +85,31 @@ status within Tigris.
...
+
+Below is the list of IAM APIs supported in Tigris. Note that IAM APIs are served
+at port `8009` for now.
```
Comment (ovaistariq) [inline docs/api/s3/index.md:91]: are served at `<entire-url-here>`

File: docs/api/s3/index.md (hunk 1 above)
Comment (JigarJoshi) [reply docs/api/s3/index.md:88]: The reason why we don't have table here is, we support very little set of IAM and CloudFront APIs. So it is easy to just list the entries we support.

File: docs/api/s3/index.md (hunk 2 above)
Comment (JigarJoshi) [reply docs/api/s3/index.md]: updated.

File: docs/api/s3/index.md (hunk 1 above)
Comment (ovaistariq) [reply docs/api/s3/index.md:88]: Yeah, I get it. You don't have to list everything. Just list the ones we support but let's have them in a table so it is consistent with the rest of the content.

Comment (JigarJoshi) [reply docs/api/s3/index.md:88]: updated.

File: docs/api/s3/index.md [hunk 3]
```diff
@@ -85,6 +85,33 @@ status within Tigris.
...
 | [UploadPartCopy](https://docs.aws.amazon.com/AmazonS3/latest/API/API_UploadPartCopy.html)                         | Yes                 |
 
+## IAM APIs
+
+Below is the list of IAM APIs supported in Tigris. Note that IAM APIs are served
+at `https://fly.storage.tigris.dev:8009` for now.
+
+| IAM API                                                                                             | Supported in Tigris |
+| --------------------------------------------------------------------------------------------------- | ------------------- |
+| - [CreateAccessKey](https://docs.aws.amazon.com/IAM/latest/APIReference/API_CreateAccessKey.html)   | Yes                 |
```
Comment (ovaistariq) [inline docs/api/s3/index.md:95]: Remove the `-` :)

**Review by ovaistariq — APPROVED** [review]

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**